Assets/MixedRealityToolkit.SDK/Features/UX/Scripts/Collections/GridObjectCollection.cs

```diff
@@ -49,7 +49,35 @@ public OrientationType OrientType
         public LayoutOrder Layout
         {
             get { return layout; }
-            set { layout = value; }
+            set 
+            { 
+                layout = value; 
+                UpdateCollection();
+            }
+        }
+
+        public enum AnchorType
```

Comment:
I'd probably pull this out and make it a separate file. I suspect will be reused for future Object Collections.